cli/src/main/java/com/liferay/blade/cli/BladeSettings.java

```diff
@@ -29,6 +29,10 @@
  */
 public class BladeSettings {
 
+	public static String getLiferayVersionDefault() {
```

Comment:
The method in BLadeSettings should just be `getLiferayVersion`.   Then internal to that method you would check for the existing setting/property and if it doesn't exist, then you would grab the default (which would be 7.1).